tests/packagedcode/data/bitbake/initramfs-live-install-testfs_1.0.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Live image install script with a second rootfs/kernel"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+SRC_URI = "file://init-install-testfs.sh"
+DEPENDS = "do_this and that"
+
+RDEPENDS_${PN} = "grub (== 12.23) parted e2fsprogs-mke2fs"
```

Comment:
I think the test should use a recipe that is a bit more complex than this one here - for me https://git.yoctoproject.org/poky/plain/meta/recipes-graphics/cairo/cairo_1.16.0.bb was always a good example